<!DOCTYPE HTML>
<!--
	Massively by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Methods</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					<header id="header">
						<a href="https://www.ncbi.nlm.nih.gov/books/NBK9932/" class="logo">More on Microtubules</a>
					</header>

				<!-- Nav -->
					<nav id="nav">
						<ul class="links">
							<li><a href="index.html">results</a></li>
							<li class="active"><a href="generic.html">methods</a></li>
							<li><a href="elements.html">additional info</a></li>
						</ul>
						<ul class="icons">
							<li><a href="" class=""><span class="label"></span></a></li>
							<!--<li><a href="#" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li>
							<li><a href="#" class="icon brands fa-instagram"><span class="label">Instagram</span></a></li>
							<li><a href="#" class="icon brands fa-github"><span class="label">GitHub</span></a></li>-->
						</ul>
					</nav>

				<!-- Main -->
					<div id="main">

						<!-- Post -->
							<section class="post">
								<header class="major">
									<!--<span class="date"></span>-->
									<h1>Computational methodology</h1>
									<p></p>
								</header>
								<div class="image main"><img src="images/pic01.jpg" alt="" /></div>
								<!--<p>Donec eget ex magna. Interdum et malesuada fames ac ante ipsum primis in faucibus. Pellentesque venenatis dolor imperdiet dolor mattis sagittis. Praesent rutrum sem diam, vitae egestas enim auctor sit amet. Pellentesque leo mauris, consectetur id ipsum sit amet, fergiat. Pellentesque in mi eu massa lacinia malesuada et a elit. Donec urna ex, lacinia in purus ac, pretium pulvinar mauris. Nunc lorem mauris, fringilla in aliquam at, euismod in lectus. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Curabitur sapien risus, commodo eget turpis at, elementum convallis enim turpis, lorem ipsum dolor sit amet nullam.</p>
								<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis dapibus rutrum facilisis. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Etiam tristique libero eu nibh porttitor fermentum. Nullam venenatis erat id vehicula viverra. Nunc ultrices eros ut ultricies condimentum. Mauris risus lacus, blandit sit amet venenatis non, bibendum vitae dolor. Nunc lorem mauris, fringilla in aliquam at, euismod in lectus. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. In non lorem sit amet elit placerat maximus. Pellentesque aliquam maximus risus. Donec eget ex magna. Interdum et malesuada fames ac ante ipsum primis in faucibus. Pellentesque venenatis dolor imperdiet dolor mattis sagittis. Praesent rutrum sem diam, vitae egestas enim auctor sit amet. Pellentesque leo mauris, consectetur id ipsum.</p>
							</section>-->
							<!-- Table -->
									<h2></h2>

									<h3></h3>
									<div class="table-wrapper">
										<table>
											<thead>
												<tr>
													<th>Methods</th>
													<th>goal/ description</th>
													<!--<th>USEFUL MATERIALS*</th>-->
												</tr>
											</thead>
											<tbody>
												<tr>
													<td>Kalign</td>
													<td>A fast and accurate multiple sequence alignment algorithm for DNA and protein sequences. This tool uses the Wu-Manber string-matching algorithm, to improve both the accuracy and speed of multiple sequence alignment. It is a fast and robust alignment method, especially well suited for the increasingly important task of aligning large numbers of sequences. (BMC Bioinformatics 6:298)  
													</td>
													<td></td>
												</tr>
												<tr>
													<td>AlphaFold</td>
													<td>AlphaFold is an AI system developed by DeepMind that predicts a protein’s 3D structure from its amino acid sequence. It regularly achieves accuracy competitive with experiment. (AlphaFold. Nature (2021))  
													</td>
													<td></td>
												</tr>
												<tr>
													<td>GROMACS</td>
													<td>GROMACS is a molecular dynamics package mainly designed for simulations of proteins, lipids, and nucleic acids. It was originally developed in the Biophysical Chemistry department of University of Groningen, and is now maintained by contributors in universities and research centers worldwide.GROMACS is one of the fastest and most popular software packages available, and can run on central processing units (CPUs) and graphics processing units (GPUs). It is free, open-source software released under the GNU General Public License (GPL), and starting with version 4.6, the GNU Lesser General Public License (LGPL). (J Comput Chem. 26 (16): 1701–18.)  
													</td>
													<td></td>
												</tr>
												<tr>
													<td>Biopython</td>
													<td>Biopython is a set of freely available tools for biological computation written in Python by an international team of developers.

It is a distributed collaborative effort to develop Python libraries and applications which address the needs of current and future work in bioinformatics. The source code is made available under the Biopython License, which is extremely liberal and compatible with almost every license in the world. (Cock et al., 2009)  
													</td>
													<td></td>
												</tr>
												<tr>
													<td>Docking</td>
													<td>Molecular docking is a key tool in structural molecular biology and computer-assisted drug design. The goal of ligand-protein docking is to predict the predominant binding mode(s) of a ligand with a protein of known three-dimensional structure. Successful docking methods search high-dimensional spaces effectively and use a scoring function that correctly ranks candidate dockings. Docking can be used to perform virtual screening on large libraries of compounds, rank the results, and propose structural hypotheses of how the ligands inhibit the target. (Morris, G.M., Lim-Wilby, M. (2008))  
													</td>
													<td></td>
													
												</tr>
												
											</tbody>
											<tfoot>
												<tr>
													<td colspan="2"></td>
													<td></td>
												</tr>
											</tfoot>
										</table>
										
										<!--<p style="font-size: 1.1rem; color: #CB3818; font-family: Gill Sans; font-style: normal"> *Certain materials may be password-protected. Contact corresponding instructor during the school for access.<p>-->
										<p style="font-size: 1.3rem; color: #3918CB; font-family: Gill Sans; font-style: normal"> *Tools/ packages listed above are open sourced & available for possible modification and redistribution.<p>
									</div>

					</div>

				<!-- Footer -->
					<footer id="footer">
						<section>
							<form method="post" action="https://formspree.io/f/mrgwzkry">
								<div class="fields">
									<div class="field">
										<label for="name">Name</label>
										<input type="text" name="name" id="name" />
									</div>
									<div class="field">
										<label for="email">Email</label>
										<input type="text" name="email" id="email" />
									</div>
									<div class="field">
										<label for="message">TO DISCUSS</label>
										<textarea name="message" id="message" rows="3"></textarea>
									</div>
								</div>
								<ul class="actions">
									<li><input type="submit" value="Send" /></li>
								</ul>
							</form>
						</section>
						<section class="split contact">
							<section class="alt">
								<h3>AUTHOR</h3>
								<p>Sivanujan Suthaharan<br>
								</p>
							</section>
							<section>
								<h3>AFFILIATION</h3>
								<p><a href="#">
Department of Chemistry<br>
								Iillinois State University<br>
								Normal, IL</a></p>
							</section>
							<section>
								<h3>EMAIL</h3>
								<p><a href="#">
ssuthah@ilstu.edu</a></p>
							</section>
							<section>
								<h3>SOCIAL</h3>
								<ul class="icons alt">
									<li><a href="www.linkedin.com/in/sivanujansuthaharan" class="icon brands alt fa-linkedin"><span class="label">Twitter</span></a></li>
									<!-- <li><a href="#" class="icon brands alt fa-facebook-f"><span class="label">Facebook</span></a></li>
									<li><a href="#" class="icon brands alt fa-instagram"><span class="label">Instagram</span></a></li>
									<li><a href="#" class="icon brands alt fa-github"><span class="label">GitHub</span></a></li>-->
								</ul>
							</section>
						</section>
					</footer>

				<!-- Copyright -->
					<div id="copyright">
						<ul><li>&copy; sivanu</li><li>Hosted via GitHub repository</a></li></ul>
					</div>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>